action.yml

```diff
@@ -14,9 +18,24 @@ runs:
       shell: bash
       run: |
         set -x
-        if snap info lxd | grep "installed"; then
-          sudo snap refresh lxd --channel=${{ inputs.channel }}
+        if [ "$(dpkg-query -f "\${db:Status-Status} \${db:Status-Eflag}" -W lxd 2>/dev/null)" = "installed ok" ]; then
```

Comment:
changed my mind on this. I don't think we should be checking dpkg or PATH - we should trust the user to use the action correctly and give them maximal power.

Reply:
I think this still should stay to avoid LXD from snap installation if it is already installed as package.
Should remove all the checks?

Reply:
The debian package can be installed locally (dpkg -i) or from 3rd party repo.

Anyway, resolving the PR as the proper fix should be addressed in [PR#5](https://github.com/canonical/setup-lxd/pull/5) (as mentioned [here](https://github.com/canonical/setup-lxd/pull/4#issuecomment-1386150693)).